pmlb/pmlb.py

```diff
@@ -256,4 +256,57 @@ def get_reviewed_datasets(dataset_names, local_cache_dir = 'datasets/'):
         if header != '# Reviewed by [your name here]\n':
             reviewed_datasets.append(dataset_name)
             
-    return sorted(reviewed_datasets)
\ No newline at end of file
+    return sorted(reviewed_datasets)
+
+def filter_datasets(obs_min = None, obs_max = None, feat_min = None, feat_max = None, class_min = None, class_max = None, endpt = None, max_imbalance = None, task = None):
+     """Filters existing datasets by given parameters, and returns a list of their names.
```

Comment:
I counted 5 spaces of indentation here. We should probably check to make sure it's the 4-space indent.